tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project/Ballerina.toml

```diff
@@ -1,5 +1,7 @@
-[project]
-org-name= "testorg"
+[package]
+org= "testorg"
+name="foo"
 version= "1.0.0"
+exported=["testpkg.foo"]
```

Comment:
Just wondering from where this `testpkg` comes, as I couldn't notice any other usages of it in the PR

Reply:
ATM other usage classes are already failing.

Reply:
I see. Cool!